src/main/java/net/snowflake/ingest/streaming/internal/ArrowRowBuffer.java

```diff
@@ -956,6 +957,7 @@ private float convertRowToArrow(
 
     // Insert nulls to the columns that doesn't show up in the input
     for (String columnName : Sets.difference(this.fields.keySet(), inputColumnNames)) {
+      rowBufferSize += 0.125; // 1/8 for null value bitmap
```

Comment:
Interesting, how'd you come across this issue?

Reply:
I just read through the code and realize we didn't add this for null values